### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by prolic**
- require PHP 7.1
- update to new prooph components

### CODE REVIEW

**Comment by prolic** [comment]
ups, some merge conflicts, one sec :)

File: composer.json [hunk 1]
```diff
@@ -18,6 +18,10 @@
     {
       "name": "Sascha-Oliver Prolic",
       "email": "saschaprolic@googlemail.com"
+    },
+    {
+      "name": "Bas Kamer",
+      "email": "bas@bushbaby.nl"
```
Comment (basz) [inline composer.json]: pls baskamer@gmail.com

File: composer.json [hunk 2]
```diff
@@ -76,5 +87,11 @@
     "test": "phpunit",
     "test-coverage": "phpunit --coverage-clover build/logs/clover.xml",
     "docs": "bookdown docs/bookdown.json"
-  }
+  },
+  "repositories": [
+    {
+      "type": "git",
+      "url": "https://github.com/prolic/prophecy.git"
```
Comment (basz) [inline composer.json:95]: temporarily ?

Comment (prolic) [reply composer.json:95]: Prophecy has currently no PHP 7.1 support.
I created a PR, but it's not yet accepted, see: https://github.com/phpspec/prophecy/pull/287

**Pull request merged**